setup_venv.ps1

```diff
@@ -4,6 +4,23 @@ if ($arguments -eq "--update-src"){
 	git pull
 }
 
+if ($arguments -eq "--force"){
+	if (Test-Path env:VIRTUAL_ENV) {
+        Write-Host "deactivating..."
+        Deactivate
+    }
+    
+    if (Test-Path .\shark.venv\) {
+        Write-Host "removing and recreating venv..."
+        Remove-Item .\shark.venv -Force -Recurse
+        if (Test-Path .\shark.venv\) {
+            Write-Host "could not remove .\shark-venv - please try running setup_venv.ps1 again!"
```

Comment:
should be
Write-Host 'could not remove .\shark-venv - please try running "setup_venv.ps1 --force" again!'